tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-cad-module/pom.xml

```diff
@@ -36,6 +36,17 @@
       <artifactId>tika-parser-microsoft-module</artifactId>
       <version>${project.version}</version>
     </dependency>
+
+    <dependency>
+    	<groupId>com.fasterxml.jackson.core</groupId>
+    	<artifactId>jackson-core</artifactId>
+	    <version>${jackson.version}</version><!--$NO-MVN-MAN-VER$-->
```

Comment:
Shouldn't need versions for jackson-core or jackson-databind because those are in dependency management in tika-parent.  I'm still in the process of cleaning these up, and there are quite a few cases where I've done exactly this.  This isn't your fault.